zulipterminal/config/keys.py

```diff
@@ -396,6 +396,12 @@ class KeyBinding(TypedDict):
         'help_text': 'Show/hide full raw message (from message information)',
         'key_category': 'msg_actions',
     }),
+    ('COPY_MESSAGE', {
+        'keys': ['C'],
+        'help_text':
+            'Copy message content to clipboard',
+        'key_category': 'msg_actions',
```

Comment:
We've discussed this in the stream, but
- `c` should not need capitalization, with some adjustment, if we stick with that hotkey, though key choice right now is not important
  - though note that we already have `c` for copying stream email address
  - plus we discussed something similar for the copy non-markdown content version
- the help_text likely doesn't need to wrap? That's only been done for long lines
- we may want the COPY_MESSAGE to be more meaningful if we have two versions :)